x-pack/plugins/reporting/server/config/create_config.test.ts

```diff
@@ -76,25 +55,28 @@ describe('Reporting server createConfig$', () => {
   });
 
   it('uses the user-provided encryption key', async () => {
-    mockInitContext = makeMockInitContext({
-      encryptionKey: 'iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii',
-      kibanaServer: {},
-    });
+    mockInitContext = coreMock.createPluginInitializerContext(
+      createMockConfigSchema({
+        encryptionKey: 'iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii',
```

Comment:
iiiiiiiiiiiiiiiiiiiiii ;)

Reply:
I'll move changes like this to a different PR :)

Ideally this PR will just be a switchover to TM and removal of ESQueue code